.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+git:
+  depth: 3
+jdk:
+  - oraclejdk8
+env:
+  - HADOOP_VERSION=2.7.5
```

Comment:
will these run in parallel or one by one ?

Reply:
parallel